static/css/style.css

```diff
@@ -689,4 +689,29 @@ input[type='date']{
 
 input[type='checkbox']{
     display: none;
-}
\ No newline at end of file
+}
+
+#filter-string{
+    text-align: center;
+    vertical-align: middle;
+    padding: 9%;
```

Comment:
Perhaps reduce to ~5% padding?
![screenshot-2](https://cloud.githubusercontent.com/assets/263363/10962077/d179a890-839d-11e5-9fbf-70c5effcb184.png)